tests/testthat/test-social-smells.R

```diff
@@ -0,0 +1,112 @@
+test_that("Using Perceval version other than 0.12.24 will fail for parse_mbox", {
+  perceval_path <- "../../../tools/bin/perceval-incompatible-version"
+  mbox_path <- "../../../../rawdata/mbox/project.mbox"
+  expect_error(parse_mbox(perceval_path, mbox_path))
+})
+test_that("Correct config file parameters and Perceval version 0.12.24 work", {
+  perceval_path <- "../../../tools/bin/perceval"
```

Comment:
Hi Carlos,
I think I meant to put "../../../tools/perceval/bin/perceval". Since in the tools.yml, as the default it uses "~/perceval/bin/perceval", and perceval is usually stored in a bin folder from what I've seen. I will keep that in mind that we shouldn't have a bin folder within the tools folder.

Reply:
I see. Yes, let's just assume all the binaries are inside a tools folder rather than a tools/bin, since I am not sure what else the tools/ folder would be used for. Kaiaulu mostly relies on the third parties that can be self-contained in a single file (e.g. a .jar or a binary), to avoid cluttering and overhead.